src/modeling/core/coefficients.jl

```diff
@@ -46,6 +46,18 @@ function evaluate_coefficient(cache::FieldCoefficientCache{T}, geometry_cache, q
     end
     return val
 end
+# GPU coefficient evaluation!
+function evaluate_coefficient(cache::FieldCoefficientCache{T}, geometry_cache::FerriteUtils.DeviceCellCache, qv::FerriteUtils.StaticQuadratureValues, t) where T
+    @unpack elementwise_data, cv = cache
+    val = zero(T)
+    cellidx = FerriteUtils.cellid(geometry_cache)
```

Comment:
I think this should be just
```suggestion
    cellidx = Ferrite.cellid(geometry_cache)
```